dora/tests/src/test/java/alluxio/stress/StressWorkerBenchIntegrationTest.java

```diff
@@ -37,8 +37,8 @@ public void readThroughput() throws Exception {
         "--base", sLocalAlluxioClusterResource.get().getMasterURI() + "/stress-worker-base/",
         "--threads", "2",
         "--file-size", "1m",
-        "--block-size", "128k",
         "--warmup", "0s", "--duration", "1s",
+        "--cluster-limit", "1"
```

Comment:
this is actually a bit awkward - In order to distribute the load among job workers, they need to know the total number of job workers and their own index. The same for --local mode (because that one commandline process needs to know it is the only process too). 

Maybe I should create a new `--worker-num` param instead of reusing this `--cluster-limit`?

Reply:
how do i get rid of this?